spring-integration-zip/src/main/java/org/springframework/integration/zip/splitter/UnZipResultSplitter.java

```diff
@@ -21,32 +21,51 @@
 import java.util.Map;
```

Comment:
One mode code changing requirement from us: the Copyring should be updated to the current year - 2015-2016 for this case. Minor, of course, but seems for me we have such a note In the Contribution Guideline